<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenetreAjouterEtudiant</class>
 <widget class="QWidget" name="FenetreAjouterEtudiant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FenetreAjouterEtudiant</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton#boutonAjouterEtudiant{
	font: 75 14pt &quot;Tw Cen MT&quot;;
	border: 1px solid #ccc;
	border-radius: 4px;
	background-color: rgb(55, 94, 173);
	color: #fff;
}

QPushButton#boutonAjouterAmphi:hover {
	background-color: rgb(47, 80, 147);
}

QMessageBox { 
	font-size: 15px;
 }

QLabel {
	font: 75 10pt &quot;MS Shell Dlg 2&quot;;
	font-weight: bold;
	text-align: center;
}

QComboBox, QSpinBox, QLineEdit{
     border: 1px solid gray;
     border-radius: 3px;
     padding: 1px 18px 1px 3px;
     min-width: 6em;
 }
 
 QComboBox:!editable, QComboBox::drop-down:editable {
      background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                  stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                  stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
 }

 QSpinBox:!editable {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                  stop: 0 #fff, stop: 0.4 #fff,
                                  stop: 0.5 #fff, stop: 1.0 #fff);
 }

QSpinBox:editable:on{
	background-color: #ffffff;
}

 QComboBox:!editable:on, QComboBox::drop-down:editable:on {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #D3D3D3, stop: 0.4 #D8D8D8,
                                 stop: 0.5 #DDDDDD, stop: 1.0 #E1E1E1);
 }
 
 QComboBox:on{
     padding-top: 3px;
     padding-left: 4px;
 }
 
 QComboBox::drop-down {
     subcontrol-origin: padding;
     subcontrol-position: top right;
     width: 15px;
 
     border-left-width: 1px;
     border-left-color: darkgray;
     border-left-style: solid;
     border-top-right-radius: 3px;
     border-bottom-right-radius: 3px;
 }

 QComboBox::down-arrow, QSpinBox::down-arrow{
     image: url(:/imgs/images/flecheBas.png);
	 width: 10px;
 }

QSpinBox::up-arrow{
	 image: url(:/imgs/images/flecheHaut.png);
	 width: 10px;
}

 QComboBox QAbstractItemView{
     border: 2px solid darkgray;
     selection-background-color: #394da7;
 }</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1" rowspan="2">
    <widget class="QGroupBox" name="groupBoxAjouterEtudiant">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>300</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QFormLayout" name="formLayout">
        <property name="horizontalSpacing">
         <number>28</number>
        </property>
        <property name="leftMargin">
         <number>42</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="labelMatricule">
          <property name="text">
           <string>Matricule de l'Etudiant :</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinBoxMatricule">
          <property name="minimumSize">
           <size>
            <width>101</width>
            <height>30</height>
           </size>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelNomEtudiant">
          <property name="text">
           <string>Nom de l'Etudiant: </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditNomEtudiant">
          <property name="minimumSize">
           <size>
            <width>101</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelPrenomEtudiant">
          <property name="text">
           <string>Prénom de l'Etudiant : </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEditPrenomEtudiant">
          <property name="minimumSize">
           <size>
            <width>101</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelDateNaissance">
          <property name="text">
           <string>Date de Naissance de l'Etudiant :</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="labelJour">
              <property name="text">
               <string>Jour</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelMois">
              <property name="text">
               <string>Mois</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelAnnee">
              <property name="text">
               <string>Année</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QSpinBox" name="spinBoxJour">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>31</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxMois">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>12</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxAnnee">
              <property name="minimum">
               <number>1900</number>
              </property>
              <property name="maximum">
               <number>999999999</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="labelAnneeAcademique">
          <property name="text">
           <string>Année Académique :</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="labelAmphi">
          <property name="text">
           <string>Amphi de l'Etudiant :</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="comboBoxAnneeAcademique">
          <property name="minimumSize">
           <size>
            <width>101</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="insertPolicy">
           <enum>QComboBox::InsertAlphabetically</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QComboBox" name="comboBoxAmphi">
          <property name="minimumSize">
           <size>
            <width>101</width>
            <height>30</height>
           </size>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>3</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="boutonAjouterEtudiant">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Ajouter</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>47</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>47</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
